changelog/5449.rst

```diff
@@ -0,0 +1,9 @@
+`sunpy.map.GenericMap.date` now looks for more metadata than just DATE-OBS.
+This property can return any one of the new properties (see above) depending
```

Comment:
```suggestion
This property can return any one of the new properties depending
```
What is above? Is this referencing the previous changelog?

Reply:
Woops, this is hungover from when I just copied and pasted from the what's new.

Reply:
Should be fixed now.